### DISCUSSION TITLE
Map DaVinci Resolve Speed Editor Jog Wheel to Scrub Through FCP Clips (Not The Project)

### DISCUSSION THREAD

**Discussion by masterbeat99** [CommandPost/FCPCafe#426] [Final Cut Pro Questions]
Hey there, forgive me if there is an easy way to do this and I'm missing it!

I do a lot of scrubbing through and auditioning clips in FCP in the clips editor (not the main timeline) to mark sections in/out points, tag them with keywords and mark them to be inserted in projects.

So far, with CommandPost and Speed Editor, I can only get the jog wheel to control the main project timeline, I can't ever seem to get it to scrub/skim through the auditioning clips in top left, even when my playhead is active there.

How can I map the Speed Editor jog wheel to scrub through those clips?  My goal is to scrub back and forth in those clips, use in/out to mark clips.  That is working, and stop/play will also control clips when I make my playhead active there, but the Jog Wheel won't cooperate.  I'm sure I'm missing something!

Thanks...

**Comment by benblaser60** [comment]
Retiming parameters are not able to be made into presets. Apple has not coded them to do that. Basically, if it's not in the Adjustments pane, it can't be a preset.
